<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWidget</class>
 <widget class="QWidget" name="GameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>24点-对战</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/logo1.png</normaloff>:/img/logo1.png</iconset>
  </property>
  <widget class="QTabWidget" name="modeSelectTabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="solveMode">
    <attribute name="title">
     <string>自动求解</string>
    </attribute>
    <widget class="QTextBrowser" name="solveModeTextShowcase">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>360</width>
       <height>140</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="solveModeTextEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>250</width>
       <height>70</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入题目:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="solveModeFileButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>选择文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="solveModeSolveButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>220</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>开始求解</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="offlineMode">
    <attribute name="title">
     <string>单人模式</string>
    </attribute>
    <widget class="QTextBrowser" name="offlineModeNumShowcase1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>75</width>
       <height>106</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="offlineModeNumShowcase2">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>20</y>
       <width>75</width>
       <height>106</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="offlineModeNumShowcase3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>75</width>
       <height>106</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="offlineModeNumShowcase4">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>20</y>
       <width>75</width>
       <height>106</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="offlineModeTextEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>265</width>
       <height>110</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入答案:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="offlineModePointShowcase">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>140</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>得分:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="offlineModeGiveupButton">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>180</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>显示答案</string>
     </property>
    </widget>
    <widget class="QPushButton" name="offlineModeCheckButton">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>220</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QLabel" name="offlineModePoker1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>75</width>
       <height>106</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="offlineModePoker2">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>20</y>
       <width>75</width>
       <height>106</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="offlineModePoker4">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>20</y>
       <width>75</width>
       <height>106</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="offlineModePoker3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>75</width>
       <height>106</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="onlineMode">
    <attribute name="title">
     <string>联机模式</string>
    </attribute>
    <widget class="QStackedWidget" name="onlineModeStackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>400</width>
       <height>270</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="onlineModeCreatPage">
      <widget class="QComboBox" name="onlineModePortSelect">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>140</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1145</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1146</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1147</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1148</string>
        </property>
       </item>
      </widget>
      <widget class="QTextEdit" name="onlineModeIDEdit">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>60</y>
         <width>80</width>
         <height>70</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>请输入昵称:</string>
       </property>
      </widget>
      <widget class="QLabel" name="onlineModeCreatLabel">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>200</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>联机大厅</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="onlineModeStartButton">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>200</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>寻找房间</string>
       </property>
      </widget>
      <widget class="QLabel" name="onlineModeDebugLabel">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>200</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>debug</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="onlineModeQuitRoomButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>75</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>离开房间</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="onlineModeMemberTabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>280</width>
         <height>190</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>用户昵称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>身份</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="onlineModeGamePage">
      <widget class="QTextBrowser" name="onlineModeNumShowcase3">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>75</width>
         <height>106</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="onlineModeNumShowcase4">
       <property name="geometry">
        <rect>
         <x>305</x>
         <y>40</y>
         <width>75</width>
         <height>106</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="onlineModeNumShowcase1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>75</width>
         <height>106</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="onlineModeNumShowcase2">
       <property name="geometry">
        <rect>
         <x>115</x>
         <y>40</y>
         <width>75</width>
         <height>106</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="onlineModeTextEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>265</width>
         <height>90</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>请输入答案:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="onlineModeCheckButton">
       <property name="geometry">
        <rect>
         <x>305</x>
         <y>200</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="onlineModePointShowcase">
       <property name="geometry">
        <rect>
         <x>305</x>
         <y>159</y>
         <width>75</width>
         <height>30</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>得分: </string>
       </property>
      </widget>
      <widget class="QLabel" name="onlineModeGameLabel">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>200</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>联机游戏中</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="onlineModeQuitGameButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>5</y>
         <width>75</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>结束游戏</string>
       </property>
      </widget>
      <widget class="QLabel" name="onlineModePoker1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>75</width>
         <height>106</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="onlineModePoker2">
       <property name="geometry">
        <rect>
         <x>115</x>
         <y>40</y>
         <width>75</width>
         <height>106</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="onlineModePoker4">
       <property name="geometry">
        <rect>
         <x>305</x>
         <y>40</y>
         <width>75</width>
         <height>106</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="onlineModePoker3">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>75</width>
         <height>106</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="onlineModeRankPage">
      <widget class="QTableWidget" name="onlineModeRankTabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>360</width>
         <height>170</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>用户昵称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>得分</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="onlineModeExitButton">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>220</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>返回大厅</string>
       </property>
      </widget>
      <widget class="QPushButton" name="onlineModeSaveButton">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>220</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>保存排名</string>
       </property>
      </widget>
      <widget class="QLabel" name="onlineModeRankLabel">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>200</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>排行榜</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
